ddtrace/llmobs/_writer.py

```diff
@@ -363,6 +363,29 @@ def dataset_pull(self, name: str) -> Dataset:
             )
         return Dataset(name, dataset_id, class_records)
 
+    def project_create(self, name: str) -> str:
```

Comment:
this end point is technically create or get, but if neither of these will be exposed in the long term, this is fine for now

Reply:
Yeah fair callout. I just wanted to separate the create/get calls into separate clean methods on the writer client side